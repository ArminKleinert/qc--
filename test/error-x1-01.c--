
register bits32 x;
register bits32 x;

foo () 
{

}
