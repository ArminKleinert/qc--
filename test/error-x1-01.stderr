file error-x1-01.c--, line 3, characters 1-19 Error: value already declared: x
